### PR TITLE
Mention PHP for exception replay and dynamic instrumentation

### PR DESCRIPTION
**Pull request by bwoebi**
### What does this PR do? What is the motivation?
With PHP 1.4 DI and exception replay are now supported.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-9211 to review

File: content/en/dynamic_instrumentation/enabling/php.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Dynamic Instrumentation is a feature of supporting Datadog tracing libraries. If you are already using [APM to collect traces][1] for your application, ensure your Agent and tracing library are on the required version, and go directly to enabling Dynamic Instrumentation in step 4.
+
+## Installation
+
+1. Install or upgrade your Agent to version [7.45.0][7] or higher.
+2. If you don't already have APM enabled, in your Agent configuration, set the `DD_APM_ENABLED` environment variable to `true` and listening to the port `8126/TCP`.
+3. Install or upgrade the PHP tracing libraries to version 1.4.0, by following the [relevant instructions][2].
+
+   **Note**: Dynamic Instrumentation is available in the `dd-trace-php` library in versions 1.4.0 and later. Currently only function and method probes are supported.
```
Comment (brett0000FF) [inline content/en/dynamic_instrumentation/enabling/php.md:24]:
```suggestion
   **Note**: Dynamic Instrumentation is available in the `dd-trace-php` library in versions 1.4.0 and later. Only function and method probes are supported.
```

**Review by brett0000FF — APPROVED** [review]
Looks good--thanks! I left one very minor suggestion to align with our style best practices.

**Review by davidejones — APPROVED** [review]

**Pull request merged**